packages/lib/default-cookies.ts

```diff
@@ -20,7 +20,7 @@ export function defaultCookies(useSecureCookies: boolean): CookiesOptions {
   const cookiePrefix = useSecureCookies ? "__Secure-" : "";
 
   const defaultOptions: CookieOption["options"] = {
-    domain: isENVDev ? undefined : NEXTAUTH_COOKIE_DOMAIN,
+    domain: isENVDev ? "cal.local" : NEXTAUTH_COOKIE_DOMAIN,
```

Comment:
Had to use a different URL than "localhost" to play with cookie sessions between different subdomains. We should make everyone use this new scheme by default, it's needed to make orgs work locally.